<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-077G3EM" ignored="0" name="Demo1" tests="12" failures="7" timestamp="2023-01-30T20:29:38 IST" time="685.346" errors="0">
  <testcase name="RingPolicy_LTBC1" time="176.309" classname="com.RingPayScripts.RingPolicy_LTBC1_Scripts"/>
  <testcase name="PlayStore_Journey" time="182.837" classname="com.RingPayScripts.PlayStore_Scripts">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_PlayStore_Journey.Bnpl_Txn_Transaction_Complete.transactionComplete(Bnpl_Txn_Transaction_Complete.java:35)
at com.RingPayScripts.PlayStore_Scripts.PlayStore_Journey(PlayStore_Scripts.java:427)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken$0.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse$0.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 20 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PlayStore_Journey -->
  <testcase name="PlayStore_Journey_Segment1" time="208.652" classname="com.RingPayScripts.PlayStore_Segment1_Scripts">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_PlayStore_Journey_Segment1.Bnpl_Txn_Transaction_Complete.transactionComplete(Bnpl_Txn_Transaction_Complete.java:35)
at com.RingPayScripts.PlayStore_Segment1_Scripts.PlayStore_Journey_Segment1(PlayStore_Segment1_Scripts.java:418)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken$0.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse$0.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 20 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PlayStore_Journey_Segment1 -->
  <testcase name="PromoCode_Journey" time="212.669" classname="com.RingPayScripts.PromoCode_Scripts">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_PromoCode_Journey.Bnpl_Txn_Transaction_Complete.transactionComplete(Bnpl_Txn_Transaction_Complete.java:35)
at com.RingPayScripts.PromoCode_Scripts.PromoCode_Journey(PromoCode_Scripts.java:454)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken$0.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse$0.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 20 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PromoCode_Journey -->
  <testcase name="MerChantQRCode_Journey" time="216.536" classname="com.RingPayScripts.MerchantQRCode_Scripts">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_MerchantQRCode_Journey.Bnpl_Txn_Transaction_Complete.transactionComplete(Bnpl_Txn_Transaction_Complete.java:34)
at com.RingPayScripts.MerchantQRCode_Scripts.MerChantQRCode_Journey(MerchantQRCode_Scripts.java:462)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken$0.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse$0.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 20 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- MerChantQRCode_Journey -->
  <testcase name="Ringpolicy_L3" time="21.97" classname="com.RingPayScripts.RingPolicy_L3_Scripts">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.business.RingPay_RingPolicy_L3.RegisterUser_Login.login_Positive(RegisterUser_Login.java:46)
at com.RingPayScripts.RingPolicy_L3_Scripts.Ringpolicy_L3(RingPolicy_L3_Scripts.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- Ringpolicy_L3 -->
  <testcase name="RingPolicy_BC1" time="171.436" classname="com.RingPayScripts.RingPolicy_BC1_Scripts"/>
  <testcase name="RingPolicy_BC2" time="169.852" classname="com.RingPayScripts.RingPolicy_BC2_Scripts"/>
  <testcase name="RingPolicy_L1_TestCase" time="172.506" classname="com.RingPayScripts.RingPolicy_L1_Scripts"/>
  <testcase name="RingPolicy_L2_TestCase" time="170.918" classname="com.RingPayScripts.RingPolicy_L2_Scripts"/>
  <testcase name="MerChantQRCodeJourney_Segment1" time="222.126" classname="com.RingPayScripts.MerchantQRCode_and_PromoCode_Scripts_Segment1">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_MerchantQRCode_Journey_Segment1.Bnpl_Txn_Transaction_Complete.transactionComplete(Bnpl_Txn_Transaction_Complete.java:35)
at com.RingPayScripts.MerchantQRCode_and_PromoCode_Scripts_Segment1.MerChantQRCodeJourney_Segment1(MerchantQRCode_and_PromoCode_Scripts_Segment1.java:453)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken$0.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse$0.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 20 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- MerChantQRCodeJourney_Segment1 -->
  <testcase name="PromoCode_Journey_Segment1" time="224.562" classname="com.RingPayScripts.MerchantQRCode_and_PromoCode_Scripts_Segment1">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_PromoCode_Journey_Segment1.Bnpl_Txn_Transaction_Complete.transactionComplete(Bnpl_Txn_Transaction_Complete.java:35)
at com.RingPayScripts.MerchantQRCode_and_PromoCode_Scripts_Segment1.PromoCode_Journey_Segment1(MerchantQRCode_and_PromoCode_Scripts_Segment1.java:868)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken$0.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse$0.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 20 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PromoCode_Journey_Segment1 -->
</testsuite> <!-- Demo1 -->
